lib/common/client/middleware/response/mhv_xml_html_errors.rb

```diff
@@ -11,10 +11,11 @@ def on_complete(env)
             return if env.success?
             return unless env.response_headers['content-type'] =~ /\b(xml|html)/
             @status = env.status.to_i
+            @body = env.body.delete('%') # strip percentages from html because Sentry uses it for interpolation
```

Comment:
Could possibly do this in SentryLogging mixin instead and help avoid this headache for others, but good enough for now.